model/src/main/proto/arguments.proto

```diff
@@ -326,3 +326,8 @@ message SurveyActivityParams {
   // The ID of the triggering exploration.
   string exploration_id = 3;
 }
+
+message IntroActivityParams {
```

Comment:
Nit: add message doc comment for consistency.

Reply:
Done